calico_containers/tests/unit/endpoint_test.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from mock import patch
+from nose_parameterized import parameterized
+from calico_ctl import endpoint
+
+
+class TestEndpoint(unittest.TestCase):
```

Comment:
Looks like Walker didn't get to actually writing any of the tests for the endpoint command calls